packages/editor-ui/src/components/Projects/ProjectMoveResourceModal.vue

```diff
@@ -21,17 +23,31 @@ const props = defineProps<{
 
 const i18n = useI18n();
 const uiStore = useUIStore();
+const toast = useToast();
 const projectsStore = useProjectsStore();
 const telemetry = useTelemetry();
 
 const projectId = ref<string | null>(null);
-const processedName = computed(() => {
-	const { name, email } = splitName(props.data.resource.homeProject?.name ?? '');
+const processedName = computed(
+	() => processProjectName(props.data.resource.homeProject?.name ?? '') ?? '',
+);
+const availableProjects = computed(() =>
+	projectsStore.projects
+		.filter((p) => p.id !== props.data.resource.homeProject?.id)
+		.sort((a, b) => a.name?.localeCompare(b.name ?? '') ?? 0),
+);
+const selectedProject = computed(() =>
+	availableProjects.value.find((p) => p.id === projectId.value),
+);
```

Comment:
@alexgrozav this is just one example of why we need the N8nSelect to be able to work with objects.

1 - create array of objects
2 - feed it to component
3 - user selects item
4 - search array for an item that the user already selected
5 - it forces us to have 2 separate variables to been able to do on simple selection